app/domain/authentication/authn_jwt/fetch_jwt_signing_key_factory.rb

```diff
@@ -0,0 +1,58 @@
+module Authentication
+  module AuthnJwt
+    # Factory that returns the interface implementation of FetchSigningKey
+    class FetchJwtSigningKeyFactory
+
+      def initialize
+        @fetch_provider_uri_signing_key = FetchProviderUriSigningKey.new(@authenticator_input)
+        @fetch_jwks_uri_signing_key = FetchJwksUriSigningKey.new(@authenticator_input)
+      end
+
+      def create(authenticator_input)
```

Comment:
please leave this signature empty and initialize the authenticator_parameters in the initialized function